proof/mock.go

```diff
@@ -49,6 +50,29 @@ func MockHeaderVerifier(header wire.BlockHeader, height uint32) error {
 	return nil
 }
 
+// MockGroupVerifier is a mock verifier which approves of all group keys.
+//
+// Group key verification usually involves having imported the group anchor
+// before verification, and many unit tests are not focused on group key
+// functionality but still use functions that require a group verifier.
+// This function is used in those cases.
+func MockGroupVerifier(groupKey *btcec.PublicKey) error {
+	return nil
+}
+
+// MockGroupAnchorVerifier is a mock verifier which approves of all group anchor
+// geneses.
+//
+// Group anchor verification usually involves accurately computing a group key,
+// and many unit tests are not focused on group key functionality but still use
+// functions that require a group anchor verifier. This function is used in
+// those cases.
+func MockGroupAnchorVerifier(gen *asset.Genesis,
+	groupKey *asset.GroupKey) error {
```

Comment:
I think this would also need the tapscript root to verify properly? Still getting through commit.